src/config.c

```diff
@@ -3141,6 +3141,15 @@ static int applyClientMaxMemoryUsage(const char **err) {
     return 1;
 }
 
+static int applyClusterSlotMigrationLogConfig(const char **err) {
+    if (server.cluster_enabled) {
+        clusterCleanupSlotMigrationLog();
```

Comment:
Need a valkey.conf update for this config.

Reply:
Added